sdk/keyvault/azure-keyvault-certificates/README.md

```diff
@@ -13,7 +13,7 @@ and other secrets
 ## Getting started
 ### Install the package
 Install [azure-keyvault-certificates][pypi_package_certificates] and
-[azure-identity][azure_identity] with [pip][pip]:
+[azure-identity][azure_identity_pip] with [pip][pip]:
```

Comment:
Nitpick: PyPI makes the package available, and pip isn't required to acquire it. So if you want a new name for this link I think `azure_identity_pypi` is more to the point (though I'd just leave it as it was). Anyway that's a nitpick 😊

Reply:
thanks, i'll go ahead and change it bc im doing other changes too